app/views/common/os_selection/_operatingsystem.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= fields_for item do |f| %>
   <%= select_f f, :medium_id, os_media, :id, :to_label,
     {:include_blank => blank_or_inherit_f(f, :medium), :selected => item.medium_id},
-    {:label => _("Media"), :disabled => os_media.empty?,
+    {:label => _("Media"), :readonly => os_media.empty?,
```

Comment:
This is the media selection, not the os selection. the file names are confusing, the os selection is actually in the `_architecture.html.erb` partial. This field also needs to change, but it isn't the one that was requested in the original issue.